src/Builder.php

```diff
@@ -149,9 +149,16 @@ public function getBehavior($slug, $strategy = self::DEFAULT_STRATEGY) {
         }
 
         if (!is_callable($strategy)) {
-            $strategy = function() use ($strategy) {
-                return $strategy;
-            };
+            if (is_array($strategy) && count($strategy) === 2 && is_object($strategy[0]) && is_string($strategy[1])) {
```

Comment:
@jonyo This is a little subjective, I could be passing and event object and something else. Also it doesn't support static calls, so why don't simply use `is_callable()`? See http://viper-7.com/ZlreM2